index.js

```diff
@@ -31,11 +37,17 @@ module.exports.sync = (filename, opts) => {
 	let dir = path.resolve(opts.cwd || '');
 	const root = path.parse(dir).root;
 
+	const filenames = [].concat(filename);
+
+	const findPathSync = files => files
+		.map(name => path.join(dir, name))
+		.find(filePath => pathExists.sync(filePath));
+
 	// eslint-disable-next-line no-constant-condition
 	while (true) {
-		const fp = path.join(dir, filename);
+		const fp = findPathSync(filenames);
```

Comment:
It's done in `findPathSync`. I recall I had to take it out of there because the linter complained about my arrow function inside a loop.